### PR TITLE
Update metric normalization

### PR DESCRIPTION
**Pull request by cleptric**
https://develop-git-statsd-unicode-escapes.sentry.dev/sdk/metrics/#normalization

closes https://github.com/getsentry/sentry-php/issues/1731

### CODE REVIEW

**Comment by cleptric** [comment]
@stayallive When you look at the linked docs, regarding Line Feed, Carriage Return and Tab.
I don't think you can use these characters in PHP. Or can you?

**Comment by stayallive** [comment]
You can right... `"\n\r\t"` (only inside `"`)?

So technically you can, but it's a stretch anyone would do it on purpose. But should also be easy to replace right? Something like `str_replace(["\n", "\r", ...], ['\n', '\r', ...], $item)` I'm thinking?

File: src/Serializer/EnvelopItems/MetricsItem.php [hunk 1]
```diff
@@ -110,4 +115,27 @@ public static function toEnvelopeItem(Event $event): string
             $statsdPayload
         );
     }
+
+    private static function replaceTagValueCharacters(string $tagValue): string
```
Comment (markushi) [inline src/Serializer/EnvelopItems/MetricsItem.php:119]: Not sure if relevant for PHP, but only relay control characters are replaced as well: https://github.com/getsentry/relay/blob/3208e3ce5b1fe4d147aa44e0e966807c256993de/relay-metrics/src/protocol.rs#L62C7-L62C20

Comment (cleptric) [reply src/Serializer/EnvelopItems/MetricsItem.php:119]: Does not apply to any SDKs, we do this in Relay only.

**Review by stayallive — APPROVED** [review]
Code wise this looks good!

**Pull request merged**